show/muxcable.py

```diff
@@ -0,0 +1,352 @@
+import json
+import sys
+
+import click
+import utilities_common.cli as clicommon
+
```

Comment:
Delete this empty line. All of these imports are technically "third-party" imports, as they are not part of the standard library and not part of this (show) package.